backend/src/main/java/com/github/abcdgames/backend/player/Player.java

```diff
@@ -0,0 +1,23 @@
+package com.github.abcdgames.backend.player;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity(name = "player")
+public class Player {
+    @Id
+    @Column(name = "id", nullable = false)
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private String id;
+
+    @Column(name = "displayname")
```

Comment:
Ich meine convention ist es, das ganze mit _ zu trennen
```suggestion
    @Column(name = "display_name")
```